zds/tutorialv2/models/models_database.py

```diff
@@ -548,6 +548,9 @@ class Meta:
     objects = PublishedContentManager()
     versioned_model = None
 
+    # sizes contain a python dict (as a string in database) with all information about file sizes
+    sizes = models.CharField('Tailles des fichiers téléchargeables', max_length=512, default='{}')
```

Comment:
Parce que pour dépasser cette limite il faudrait plusieurs dizaines de formats avec plusieurs Go, To voire plus à chaque fois. Je peux augmenter si tu veux mais je n'en vois pas l'utilité.